src/commands/deployment/correlate.ts

```diff
@@ -0,0 +1,182 @@
+import chalk from 'chalk'
+import {Command, Option} from 'clipanion'
+import simpleGit from 'simple-git'
+
+import {getCISpanTags} from '../../helpers/ci'
+import {gitRepositoryURL, gitLocalCommitShas, gitCurrentBranch} from '../../helpers/git/get-git-data'
+import {Logger, LogLevel} from '../../helpers/logger'
+import {retryRequest} from '../../helpers/retry'
+import {CI_PROVIDER_NAME, CI_ENV_VARS, GIT_REPOSITORY_URL, GIT_SHA} from '../../helpers/tags'
+import {getApiHostForSite, getRequestBuilder} from '../../helpers/utils'
+
+/**
+ * This command is a wrapper around the datadog-ci tag command, allowing customers to mark CI jobs
```

Comment:
is it really a wrapper of the datadog-ci tag command?

Reply:
is it just a wrong copy paste?

Reply:
Wrong copy paste 😬